e2e/data/smoke/rook_api_external.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: rook-api-external
+  namespace: rook
+  labels:
+    app: rook-api
+    rook_cluster: rook
+spec:
+  ports:
+  - name: rook-api
+    port: 8124
+    protocol: TCP
+    targetPort: 8124
```

Comment:
Nit: if the `targetPort` is the same as `port`, you can omit `targetPort`. By default kubernetes will use `port` when targetPort is not defined.